sources/packages/backend/libs/test-utils/src/factories/program-year.ts

```diff
@@ -14,5 +15,6 @@ export function createFakeProgramYear(programYearPrefix?: number): ProgramYear {
   programYear.parentFormName = `${programYear.formName}-parent`;
   programYear.partnerFormName = `${programYear.formName}-partner`;
   programYear.programYearPrefix = programYearPrefix.toString();
+  programYear.maxLifetimeBCLoanAmount = faker.random.number(100000);
```

Comment:
I believe we can give an actual number rather giving a random number. As during the lifetime maximums calculations, if this is created, it will be helpful to do the expect.

Reply:
It will not always be 50,000 for future program years I guess. but I got the point, I will change